bigchaindb/config_utils.py

```diff
@@ -91,7 +91,11 @@ def file_config(filename=None):
               file at CONFIG_DEFAULT_PATH, if filename == None)
     """
     logger.debug('On entry into file_config(), filename = {}'.format(filename))
-    if not filename:
+
```

Comment:
Just reviewed my code, that part was smelling. I removed that thing. It was to support `autoconfigure` but without loading the default file configuration.